tasks/configure-FreeBSD.yml

```diff
@@ -0,0 +1,5 @@
+---
+- sysrc: name=varnishd_config value={{ varnish_config_path }}/default.vcl
```

Comment:
To match the style elsewhere, can you add a `name` to each of these and add a newline between each task? It's hard to tell at a glance what each change is doing.

Alternatively, you could do one task, with `with_items` and a dict to do all the changes in one task (instead of having four separate tasks).